.gitignore

```diff
@@ -5,3 +5,4 @@ count.out
 test
 profile.out
 tmp.out
+.idea/*
```

Comment:
revert this item.

Reply:
@appleboy Done and pushed.

Also noticed that this line has taken a hit the code coverage as well.

**Two questions** 

1. Any particular reason why .gitignore is not being ignored from code coverage ?
2. Any particular reason why IDE specific files should not be under the .gitignore of this repository?

Reply:
The .gitignore file only ignores the related list of the project, not your personal file like `.idea`.